src/pages/SearchPage.js

```diff
@@ -171,7 +171,7 @@ class SearchPage extends Component {
         return (
             <ScreenWrapper
                 includeSafeAreaPaddingBottom={false}
-                testID="SearchPage"
+                testID={SearchPage.displayName}
```

Comment:
This is class component so `displayName` cannot be set according to lint rule:
`Class components must not set the displayName property.` (rulesdir/display-name-property)

So `testID={undefined}` for now and showing console error:
`Failed prop type: The prop testID is marked as required in ScreenWrapper, but it's value is undefined`

@roryabraham Which option do you suggest for class components?
1. revert back to hardcoded name - `testID="SearchPage"
2. make `testID` optional
3. remove `displayName` lint rule in class component
4. Any better option?

Reply:
> remove displayName lint rule in class component

This one. Let's set `displayName` everywhere. It's more consistent and also class components are going to soon be a thing of the past so I don't think we need to optimize the DX for them so much.

Reply:
Then maybe we need to revert lint rule that every component (both function and class) requires displayName?
This way, we won't miss any component missing testID and console error won't happen again.
i.e. https://github.com/Expensify/App/issues/28334